docker_run.sh

```diff
@@ -1 +1,46 @@
-docker run -it --rm --volume $(pwd)/results:/code/results --volume $(pwd)/map:/code/map grinder-framework -h
\ No newline at end of file
+#!/usr/bin/env sh
+
+usage() {
+	echo "Usage: $(basename $0) [--cpus <quantity-of-cpu>] [-h | --help]"
+}
+
+user_cpu_count=1
+cpu_count=0
+pwd=$(pwd)
+
+while [ $# -gt 0 ]; do
+	case "$1" in
+		"-h"|"--help") usage; exit 0 ;;
+		"--cpus") user_cpu_count=$2; shift 2 ;;
+		*) usage; exit 0 ;;
+	esac
+done
+
+case "$(uname)" in
+	"Linux") cpu_count=$(grep -c 'processor' /proc/cpuinfo) ;;
+	"Darwin") cpu_count=$(sysctl -n hw.logicalcpu) ;;
+	*) cpu_count=1 ;;
+esac
+
+if [ $user_cpu_count -gt $cpu_count || $cpu_count -eq 1 ]; then
+	user_cpu_count=$cpu_count
+fi
+
+if [ $user_cpu_count -eq 0 ]; then
+	user_cpu_count=$(echo "$cpu_count / 2" | bc)
+fi
+
+if [ ! -e database.db ]; then
+	touch ./database.db
+fi
+
+mkdir -p ./results/
+mkdir -p ./map/static/data/
+
+docker run -it --rm \
+	-p 5000:5000 \
+	--cpus="$user_cpu_count" \
+	--volume ${pwd}/database.db:/app/database.db \
+	--volume ${pwd}/results:/app/results \
+	--volume ${pwd}/map/static/data:/app/map/static/data \
+	grinder-framework
```

Comment:
Maybe put it in separated `docker-compose.yml`?
https://docs.docker.com/compose/compose-file/

With basic skeleton structure like this:
- .dockerignore
- Dockerfile
- docker-compose.yml

To avoid a lot of different keys in docker run

Reply:
I will work on it.